tests/Unit/Type/ItemPriceTest.php

```diff
@@ -836,12 +925,61 @@ public function taxesProvider()
             [
                 false,
                 [
-                    [new TaxPrice(10, 'exclusive'), new TaxPrice(15, 'exclusive')],
-                    [new TaxPrice(25, 'exclusive')]
+                    [new TaxPrice(10, TaxPrice::EXCLUSIVE), new TaxPrice(15, TaxPrice::EXCLUSIVE)],
+                    [new TaxPrice(25, TaxPrice::EXCLUSIVE)]
                 ],
                 2,
                 3
             ]
         ];
     }
+
+    /**
+     * @covers ::resetTaxes
+     * @uses Blesta\Pricing\Type\ItemPrice::__construct
+     * @uses Blesta\Pricing\Type\ItemPrice::resetDiscountSubtotal
+     * @uses Blesta\Pricing\Type\ItemPrice::excludeTax
+     * @uses Blesta\Pricing\Type\ItemPrice::subtotal
+     * @uses Blesta\Pricing\Type\UnitPrice::__construct
+     * @uses Blesta\Pricing\Type\UnitPrice::setPrice
+     * @uses Blesta\Pricing\Type\UnitPrice::setQty
+     * @uses Blesta\Pricing\Type\UnitPrice::setKey
+     * @uses Blesta\Pricing\Type\UnitPrice::total
+     * @uses Blesta\Pricing\Modifier\AbstractPriceModifier::type
+     */
+    public function testResetTaxes()
+    {
+        $item = new ItemPrice(10);
+        $temp_item = new ItemPrice(10);
+
+        $item->excludeTax(TaxPrice::EXCLUSIVE);
+        $item->resetTaxes();
+        $this->assertEquals($item, $temp_item);
+    }
+
+    /**
+     * @covers ::excludeTax
+     * @uses Blesta\Pricing\Type\ItemPrice::__construct
+     * @uses Blesta\Pricing\Type\ItemPrice::resetDiscountSubtotal
+     * @uses Blesta\Pricing\Type\ItemPrice::subtotal
+     * @uses Blesta\Pricing\Type\UnitPrice::__construct
+     * @uses Blesta\Pricing\Type\UnitPrice::setPrice
+     * @uses Blesta\Pricing\Type\UnitPrice::setQty
+     * @uses Blesta\Pricing\Type\UnitPrice::setKey
+     * @uses Blesta\Pricing\Type\UnitPrice::total
+     * @uses Blesta\Pricing\Modifier\AbstractPriceModifier::type
+     */
+    public function testExcludeTax()
+    {
+        $item = new ItemPrice(10);
+        $temp_item = clone $item;
+
+        $item->excludeTax('invalid_tax_type');
+        $this->assertEquals($temp_item, $item);
```

Comment:
I think verifying the attribute changed as expected is a better test than comparing whether two objects are equal after you change the state.